include/alpaka/workdiv/WorkDivMembers.hpp

```diff
@@ -83,6 +96,14 @@ namespace alpaka
         Vec<TDim, TIdx> m_threadElemExtent;
     };
 
+    //! Deduction guide for the constructor which can be called without explicit template type parameters
+    ALPAKA_NO_HOST_ACC_WARNING
+    template<typename TDim, typename TIdx>
+    ALPAKA_FN_HOST_ACC explicit WorkDivMembers(
```

Comment:
Why is the deduction guide explicit? I would make it implicit, the same way as the constructor.
```suggestion
    ALPAKA_FN_HOST_ACC WorkDivMembers(
```